docs/content/Getting Started Using.md

```diff
@@ -32,38 +33,54 @@ At the time of writing, the [G-Research analyzers](https://github.com/g-research
+This imports two targets to your project file: `AnalyzeFSharpProject` and `AnalyzeFSharpProjectUsingFscArgs`.  
+These will allow us to easily run the analyzers for our project.  
+
+Before we can run `dotnet msbuild /t:AnalyzeFSharpProject`, we need to add specify our settings in a property called `FSharpAnalyzersOtherFlags`:
+
+```xml
+<PropertyGroup>
+    <FSharpAnalyzersOtherFlags>--analyzers-path &quot;$(PkgG-Research_FSharp_Analyzers)/analyzers/dotnet/fs&quot; --report &quot;$(MSBuildProjectName)-$(TargetFramework).sarif&quot; --treat-as-warning IONIDE-004 --verbose</FSharpAnalyzersOtherFlags>
+</PropertyGroup>
+```
 
 To locate the analyzer DLLs in the filesystem, we use the variable `$(PkgG-Research_FSharp_Analyzers)`. It's produced by NuGet and normalized to be usable by [MSBuild](https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty).
 In general, a `Pkg` prefix is added and dots in the package ID are replaced by underscores. But make sure to look at the [nuget.g.props](https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#restore-outputs) file in the `obj` folder for the exact string.  
-The `\analyzers\dotnet\fs` subpath is a convention analyzer authors should follow when creating their packages.
+The `/analyzers/dotnet/fs` subpath is a convention analyzer authors should follow when creating their packages.
 
 At last, you can run the analyzer from the project folder:
 
 ```shell
-dotnet msbuild /t:AnalyzeProject
+dotnet msbuild /t:AnalyzeFSharpProject
 ```
 
+Note: if your project has multiple `TargetFrameworks` the tool will be invoke for each target framework.
+
+### Analyze FSharp project using FscArgs
+
+So 🤔, what is the difference between `AnalyzeFSharpProject` and `AnalyzeFSharpProjectUsingFscArgs`?
+
+The way the analyzers work is that we will programmatically type-check a project and process the results with our analyzers. In order to do this programmatically we need to construct the [FSharpProjectOptions](https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-codeanalysis-fsharpprojectoptions.html).  
+This is essentially a type that represents all the fsharp compiler arguments. When using `--project`, we will use [ProjInfo](https://github.com/ionide/proj-info) to invoke a set of MSBuild targets in the project to perform a design-time build.  
+A design-time build is basically an empty invocation of a build. It won't produce assemblies but will have constructed the correct arguments to theoretically invoke the compiler.  
+
+There's an alternative way to do this. Instead of using the `--project` argument, it's possible to use the `--fsc-args` argument to let the CLI tool construct the needed `FSharpProjectOptions`.  
+The `AnalyzeFSharpProjectUsingFscArgs` uses MSBuild to construct the raw `fsc` arguments. This potentially be faster (due to MSBuild caching) and more accurate.
```

Comment:
```suggestion
The `AnalyzeFSharpProjectUsingFscArgs` uses MSBuild to construct the raw `fsc` arguments. This can potentially be faster (due to MSBuild caching) and more accurate.
```